sark/data.py

```diff
@@ -111,6 +116,30 @@ def collector(ea, fpos, original, patched):
     return patched_bytes
 
 
+def apply_patches(patched_path=None):
+    to_patch = idaapi.get_input_file_path()
```

Comment:
In a case of no input file - indicative IO exception will raised
In a case of additional binary loading - only the base binary (which the IDB was created for) path will be returned from idaapi.get_input_file_path, which is reasonable.

I don't see a problem here